### ISSUE TITLE
Excuse me, finetune, the value of the loss function is very different. Why?

### ISSUE THREAD

**Issue by Mark-sloan** [yuanyuanli85/tf-hrnet#5]
INFO:tensorflow:Restoring parameters from ../models\model.ckpt-459999
INFO:tensorflow:restore from ../models\model.ckpt-459999
INFO:tensorflow:loss = 4.0634e+00
INFO:tensorflow:acc_top1 = 5.0553e-01
INFO:tensorflow:acc_top5 = 8.7798e-01
INFO:tensorflow:l2_loss = 4.1757e-01
INFO:tensorflow:Restoring parameters from ../models\model.ckpt-469999
INFO:tensorflow:restore from ../models\model.ckpt-469999
INFO:tensorflow:loss = 4.4642e+00
INFO:tensorflow:acc_top1 = 4.9617e-01
INFO:tensorflow:acc_top5 = 8.2908e-01
INFO:tensorflow:l2_loss = 4.1753e-01
INFO:tensorflow:Restoring parameters from ../models\model.ckpt-479999
INFO:tensorflow:restore from ../models\model.ckpt-479999
INFO:tensorflow:loss = 4.4951e+00
INFO:tensorflow:acc_top1 = 5.0213e-01
INFO:tensorflow:acc_top5 = 8.2866e-01
INFO:tensorflow:l2_loss = 4.1749e-01
INFO:tensorflow:Restoring parameters from ../models\model.ckpt-489999
INFO:tensorflow:restore from ../models\model.ckpt-489999
INFO:tensorflow:loss = 4.3527e+00
INFO:tensorflow:acc_top1 = 5.0085e-01
INFO:tensorflow:acc_top5 = 8.4651e-01
INFO:tensorflow:l2_loss = 4.1745e-01
INFO:tensorflow:iter #500000: lr = 6.2500e-06 | loss = 8.8542e-01 | acc_top1 = 7.5000e-01 | acc_top5 = 1.0000e+00 | l2_loss = 4.1741e-01 | speed = 72.38 pics / sec

**Comment by yuanyuanli85** [comment]
From the log, looks like the training `loss = 8.8542e-01` is reasonable. While the validation loss is quite large over than 4.0.  I think it is better to check dataset used in eval.
